### ISSUE TITLE
Accessibility Bug: When the ‘Open Dialog’ is activated, ‘popover Example’ is opened and no announcement is made when the input field within the popover is accessed

### ISSUE THREAD

**Issue by puru-hk** [SAP/fundamental-ngx#3143]
Fundamental core component: "Popover"

Description: When the ‘Open Dialog’ is activated, ‘popover Example’ is opened and no announcement is made when the input field within the popover is accessed

Expected: When the input field within the popover is accessed, the UI type should be conveyed.

Attached screen shot:

![image](https://user-images.githubusercontent.com/32538291/91651452-7ff00e00-eaaa-11ea-939f-dc5d982d844a.png)

**Comment by Sanjay-1312** [comment]
![image](https://user-images.githubusercontent.com/78714518/119811156-675f3800-bf04-11eb-8d9e-446967706834.png)
![image](https://user-images.githubusercontent.com/78714518/119811190-70500980-bf04-11eb-83e2-83634b9ebdcf.png)
![image](https://user-images.githubusercontent.com/78714518/119811206-75ad5400-bf04-11eb-8666-9822783dbb54.png)

Hello Colleagues,

The issue still persists.
On activating the 'Open Dialog' button, the dialog appears and dialog name is not read.
On activating the 'Click me' button, the pop up appears and the pop message should have focus and should be read.
The name of the UI is not read and  no tool tip for the 'Close' button.
So please fix the issue .

Thanks and Regards
Sanjay

**Comment by Lokanathan-k** [comment]
hi Sanjay,

the netlify URL is not updated with the latest build can u test it in the below URL.

https://deploy-preview-5464--fundamental-ngx.netlify.app/#/core/popover

**Comment by Sanjay-1312** [comment]
Hello Colleagues,

The issue persists the dialog name is read twice and for close button tool tip is not provided.
On activating the 'Click Me' button the pop up appears and no information is read.
So please fix the issue.
The pop should receive focus and should be read to the user.
URL: https://deploy-preview-5464--fundamental-ngx.netlify.app/#/core/popover
JAWS 2021
Screenshot:
![image](https://user-images.githubusercontent.com/78714518/119929142-988a4780-bf9a-11eb-87b8-62adb07cf241.png)
![image](https://user-images.githubusercontent.com/78714518/119929156-9fb15580-bf9a-11eb-94f7-f45bb158ba91.png)

**Comment by sKudum** [comment]
Hi @puru-hk , in above example values in dropdown not get read by screen readers, neither on mouse over nor on keyboard access, don't we have to consider it to?

**Comment by puru-hk** [comment]
Yes we need to consider all the UI element in the page. All the UI element should read by the screen reader using keyboard.

**Comment by Sanjay-1312** [comment]
Issue Fixed
